README.md

```diff
@@ -0,0 +1,28 @@
+# COS Alerter
+
+## Configuring Alertmanager
+
+Add the following sections to your Alertmanager config to integrate with COS Alerter:
+```yaml
+receivers:
+...
+- name: cos-alerter
+  webhook_configs:
+  - url: http://192.168.122.118:5000/alive
+route:
+  ...
+  routes:
+  ...
+  - matchers:
+    - alertname = Watchdog
+    receiver: cos-alerter
+    group_wait: 0s
+    group_interval: 1m
+    repeat_interval: 1m
+```
+
```

Comment:
**suggestion:**
We should probably provide/link to:
- a sample Watchdog alert rule
- a sample alert rule for monitoring COS Alerter itself

Reply:
See #20